getting-started/data-node-authentication.md

```diff
@@ -0,0 +1,89 @@
+# Data Node Authentication [](distdb-auth)
+
+In a multi-node environment, it is necessary to ensure that the access
+node have the necessary roles and permissions on all the data nodes
+where distributed hypertables are placed.
+
+This is critical since when TimescaleDB is executing a query or
+command it will use the same role on the data nodes as on the access
+node. This means that the same roles need to exist on the data nodes
+and the access node need to have authentication set up for those
+roles.
+
+In a typical setup, you will create the distributed hypertable in a
+database and use the same database on all the data nodes. This means
+that you need to ensure that a user that is connected to the access
+node can access the same databases on the data nodes.
+
+> NOTE: Setting up a secure system is a complex task and this section
+> should not be read as recommending any particular security measures
+> for securing your system. The chapter contains technical
+> instructions for how to set up the different authentication methods.
+
+## Setting up Password Authentication [](distdb-auth.pass)
+
+The easiest authentication to set up is password authentication. It is
+not as strong as certificate authentication, but can be used for an
+initial setup or when you have good perimeter security. In this
+section, the focus is on setting up SCRAM SHA-256 password
+authentication. For other password authentication methods, [see the
+PostgreSQL manual][auth-password].
+
+> NOTE: The `passwd` method should not be used since it sends the
+> password in cleartext, and the `md5` is using a cryptographic hash
+> function that is no longer considered secure.
+
+### Setting up Password Authentication on the Data Node [](distdb-auth.pass.data-node)
+
+In order to set up password authentication for a user `new_user`, it
+is necessary to update the HBA (Host-Based Authentication) file on the
+data nodes to enable password authentication for `new_user`. The
+default name of the HBA file is `pg_hba.conf`. Details of [the
+`pg_hba.conf` format can be found in the PostgreSQL
+manual.][postgresql-hba].
+
+In this case, you want to give `new_user` in the `example.com` domain
+access to all databases on the data node, so add the following line to
+the `pg_hba.conf`.
+
+```
+host    all    new_user    .example.com    scram-sha-256
+```
+
+Once you have set up the authentication method on the data node, you
+need to check the configuration file (normally `postgresql.conf` in
+the data directory of the data node server) and make sure that the
+`hba_file` option is correctly set.
+
+### Setting up Passwords on the Access Node [](distdb-auth.pass.access-node)
+
+Passwords are not passed through the command line, but are added to a special
```

Comment:
I suggest starting with more context:
`
To securely execute queries across data nodes, the access node needs to connect to data nodes as the user executing the query on the access node, preferably with a unique password. The access node therefore needs to obtain the password to use for each user. The necessary user-password mapping is currently stored in a password file on the access node. Note that the password for a user need not be the same as the password used by the client to authenticate with the access node. But it _must_ be the same as the user's password on the data nodes.
`

Reply:
Did something along these lines. PTAL.